src/hotspot/share/gc/g1/g1VMOperations.hpp

```diff
@@ -87,18 +87,34 @@ class VM_G1CollectForAllocation : public VM_CollectForAllocation {
 };
 
 // Concurrent G1 stop-the-world operations such as remark and cleanup.
-class VM_G1Concurrent : public VM_Operation {
-  VoidClosure* _cl;
-  const char*  _message;
+class VM_G1PauseConcurrent : public VM_Operation {
+private:
```

Comment:
This `private` visibility specifier could be removed.

Reply:
Done.